backend/src/schema/resolvers/helpers/databaseLogger.ts

```diff
@@ -1,21 +0,0 @@
-/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
We're never going to log database calls anymore?

Reply:
We have zero logging - we need to change this. But his half baked debug capability utilizing an package not even in our `package.json` is more trouble then worth to keep it. Just my opinion.